src/Sulu/Bundle/AudienceTargetingBundle/composer.json

```diff
@@ -0,0 +1,36 @@
+{
+    "name": "sulu/audience-targeting",
```

Comment:
We are actually not adding this file anymore to newer bundles, because they get outdated anyway. Might make sense again if we manage to make a subtree split one day.